docs/README.md

```diff
@@ -1,39 +1,34 @@
 ## Technical Overview
 
-Kubernetes allows for extensions to its architecture in the form of custom
-resources: <https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/>.
-KubeVirt represents virtual machines as custom resources and manages changes
-to libvirt domains based on the state of those resources.
+Kubernetes allows for extensions to its architecture via *custom resources*, which adds a new endpoint in the Kubernetes API that stores and retrieves a collection API objects of a certain kind. However, the *custom resources* by themselves only enable store and retrieve structured data. To add business logic and specific functionality into Kubernetes, it is necessary to use *custom controllers*, which are clients of the Kubernetes API-Server that typically read an object's `.spec`, possibly do things, and then update the object's `.status`.
```

Comment:
Reviewing could be simpler for me if you split this to multiple commits, one for obvious corrections, another for more contested style changes.

For example, I love HREFs, and would like to see them used instead of spelling out the URL

Reply:
Thanks! It makes sense, I split to multiple commits and added HREFs